src/app/children/child-details/child-details.component.ts

```diff
@@ -15,21 +15,35 @@
  *     along with ndb-core.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import {Component, OnInit} from '@angular/core';
+import {Component, Inject} from '@angular/core';
 import {Child} from '../child';
 import {EntityMapperService} from '../../entity/entity-mapper.service';
 import {Gender} from '../Gender';
-import {ActivatedRoute} from '@angular/router';
+import {ActivatedRoute, Router} from '@angular/router';
+import {FormBuilder, FormGroup, Validators} from "@angular/forms";
+
+declare let require;
 
 @Component({
   selector: 'app-child-details',
   templateUrl: './child-details.component.html',
   styleUrls: ['./child-details.component.css']
 })
-export class ChildDetailsComponent implements OnInit {
-  child = new Child('test');
-  editable = false;   // editable schould be enabled by default and be activated by button
+export class ChildDetailsComponent {
+
+
+  child: Child = new Child("testID");
```

Comment:
creating the new Child here is irrelevant, as it will be created in the constructor properly, isn't it?

Reply:
It will throw an null pointer error when you leave child undefinded so this was the easiest workaround.

Reply:
ok, fair point - I think I have done the same in other places ...
Shall we make it a convention to use `new Child('')` in such cases?